runtime/src/blockhash_queue.rs

```diff
@@ -114,6 +114,17 @@ impl BlockhashQueue {
         }
         None
     }
+
+    pub fn get_recent_blockhashes(&self, count: usize) -> Vec<Hash> {
```

Comment:
This should return an `impl Iterator` instead of a `Vec`

Reply:
On second thought. Hold off. That `recent_blockhashes` doesn't look to be the Dequeue it used to be.  Surprised to see the need to sort there.

Reply:
So why does this have a `count` parameter? It's not used by anything other than the test. And if it wasn't there, you could return an unordered `impl Iterator<Item = &Hash>` with effectively zero cost.

Reply:
`count` was an attempt to shortcircuit the last iteration and avoid another to truncate in the caller.  Returning an `impl Iterator<Item=&Hash` though, I can move all of that logic to the caller (and avoid the hash copies)